.github/workflows/js-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: npm ci
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v2
```

Comment:
Different packages may use different build/lint rules, better to manage CI in a mono repo way using tools like `lerna`.
a reference setup from mina: https://github.com/ChainSafe/mina-rs/blob/main/.github/workflows/build-and-test.yml#L215

Reply:
I like it, thanks!